integrations/airflow/marquez_airflow/extractors/bigquery_extractor.py

```diff
@@ -29,22 +31,66 @@
     DbColumn
 )
 from marquez_airflow.extractors.sql.experimental.parser import SqlParser
+from marquez_airflow.schema import GITHUB_LOCATION
 from marquez_airflow.utils import (
     get_job_name
 )
 
 # BIGQUERY DAGs doesn't use this.
 # Required to pass marquez server validation.
+from openlineage.facet import BaseFacet
+
 _BIGQUERY_CONN_URL = \
     'jdbc:bigquery://https://www.googleapis.com/bigquery/v2:443'
 
```

Comment:
Based on the naming spec at https://github.com/OpenLineage/OpenLineage/pull/31/files , I think the namespace for BigQuery tables is just `bigquery`